src/bacnet/datalink/dlenv.c

```diff
@@ -31,7 +31,9 @@
 #include "bacnet/datalink/datalink.h"
 #include "bacnet/datalink/dlenv.h"
 #include "bacnet/datalink/dlmstp.h"
+#ifndef _MSC_VER
```

Comment:
Maybe we should use `#ifdef BACFILE` here if no file object is needed for the application?
`_MSC_VER` should only be used where there are compiler differences, and hopefully only happens in platform.h file.